### PR TITLE
Create logic files (milestone 4)

### PR DESCRIPTION
**Pull request by AlfredoElizarraras**
### This pull contains the following:
- [x] Calculate file.
- [x] Operate file.
- [x] Constants file.
- [x] Wait to import calculate file into the app file because linter issues.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        } else if (total) {
+          returnValue = {
+            total: operate(total, '0.01', operations.MULTIPLY),
+            next,
+            operation,
+          };
+        } else {
+          assignSameValues();
+        }
+        break;
```
Comment (Torres-ssf) [inline src/logic/calculate.js]: - [x] Great job on the `calculate.js` file. But the instructions require that the modifier `operation (%)` should be deal with at the `operate.js`
Screenshot from 2020-11-03 23-09-51

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @AlfredoElizarraras :hand: 
Awesome job so far on this milestone. But we need to make some changes before moving on.

- [x] The linter.yml file on your repository is from the javascript module. We need to change it for the [React module instead](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml).

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Comment by AlfredoElizarraras** [comment]
Hello @Torres-ssf :wave: 

### Thanks for your review :pray: 
**I will be applying the required changes ASAP** :raised_hands:

**Review by whiz25 — APPROVED** [review]
## Approved :clap:
Good job making the requested changes. You may merge and proceed to the next milestone.

### Martin Afani

**Comment by AlfredoElizarraras** [comment]
Hello @whiz25 :wave: 

### Thanks for your review and your approval :handshake: 
 **I will proceed to merge this changes.** :raised_hands:

**Pull request merged**